src/components/countries/Countries.js

```diff
@@ -0,0 +1,72 @@
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import europeMap from '../../assets/map_of_Europe.svg.png';
+import asiaMap from '../../assets/map_Asia.svg.png';
+import africaMap from '../../assets/map-Africa.svg.png';
+import oceaniaMap from '../../assets/map_Oceania.svg.png';
+import northAmericaMap from '../../assets/map-North_America-Subdivisions.svg.png';
+import centralAmericaMap from '../../assets/map_America_Central-color.svg.png';
+import southAmericaMap from '../../assets/map_of_South_America.svg.png';
+import './countries.css';
+import DisplayCountries from './DisplayCountries';
+
+const Countries = ({
```

Comment:
- [ ] As stated in the project requirements, the **`Details`** (here Countries) page should have a 'Back' button (<) which navigates the user to the **`Homepage`**. Kindly add this button as contained in the design template.